### ISSUE TITLE
Blue buttons do not have a visible focus ring

### ISSUE THREAD

**Issue by agg23** [cypress-io/cypress#16094]
### Current behavior
Focus rings on `Button` with `color="blue"` results in a ring that is nearly invisible.

<img width="155" alt="Screen Shot 2021-04-20 at 7 58 30 AM" src="https://user-images.githubusercontent.com/238679/115418251-33c71a80-a1ae-11eb-9d73-775cee4fb48c.png">

### Desired behavior
Focus rings should be visible on blue buttons
